src/request.rs

```diff
@@ -152,25 +162,32 @@ pub enum Request {
     TransferCoins {
         destination: XorName,
         amount: Coins,
-        transaction_id: u64, // TODO: Use the trait UUID
+        transaction_id: TransactionId,
     },
     /// Get transaction
     GetTransaction {
         coins_balance_id: XorName,
-        transaction_id: u64, // TODO: Use the trait UUID
+        transaction_id: TransactionId,
     },
     /// Get current wallet balance
     GetBalance,
     /// Create a new coin balance
     CreateCoinBalance {
         new_balance_owner: PublicKey,
         amount: Coins,
-        transaction_id: u64, // TODO: Use the trait UUID
+        transaction_id: TransactionId,
     },
     //
     // ===== Account =====
     //
-    PutAccount {
+    CreateAccount(NewAccount),
```

Comment:
Why have a new type (`NewAccount`) just for this `CreateAccount` variant, but not for `CreateAccountFor` (same amount of fields)?

Reply:
> new type (NewAccount) just for this CreateAccount variant

It's actually included as part of `CreateAccountFor` (and now `UpdateAccount` too) :slightly_smiling_face: